app/views/static/index.html.haml

```diff
@@ -1,4 +1,4 @@
-- set_meta_tags description: 'Get structured data out of the web. Code collaboration through GitHub. Run your scrapers in the cloud.'
+- set_meta_tags description: "Hassle-free web scraping. Over #{Scraper.all.size.round(-3)} free public scrapers, or write your own in Ruby, Python, PHP, Perl or Node. Push to the cloud and then set and forget."
```

Comment:
There's already a [method for this](https://github.com/openaustralia/morph/blob/master/app/models/scraper.rb#L59-L63) so I think we should use that instead of reimplementing it.